src/lib/styles/globals/controls.css

```
@@ -0,0 +1,417 @@
+    background-color var(--motion-duration-standard) var(--motion-ease-standard),
+    border-color var(--motion-duration-standard) var(--motion-ease-standard),
+    box-shadow var(--motion-duration-standard) var(--motion-ease-standard),
+    text-shadow var(--motion-duration-standard) var(--motion-ease-standard),
+    opacity var(--motion-duration-standard) var(--motion-ease-standard);
+  background: transparent;
+}
+
+.lattice-btn--primary {
+  --lattice-btn-tone: var(--accent-cyan);
+  --lattice-btn-tone-rgb: var(--rgb-running);
+  border: 1px solid var(--accent-cyan);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)